src/renderer/views/Hashtag/Hashtag.js

```diff
@@ -0,0 +1,99 @@
+import { defineComponent } from 'vue'
+import FtCard from '../../components/ft-card/ft-card.vue'
+import FtElementList from '../../components/ft-element-list/ft-element-list.vue'
+import FtFlexBox from '../../components/ft-flex-box/ft-flex-box.vue'
+import FtLoader from '../../components/ft-loader/ft-loader.vue'
+import packageDetails from '../../../../package.json'
+import { getHashtagLocal, parseLocalListVideo } from '../../helpers/api/local'
+import { isNullOrEmpty } from '../../helpers/utils'
+export default defineComponent({
+  name: 'Hashtag',
+  components: {
+    'ft-card': FtCard,
+    'ft-element-list': FtElementList,
+    'ft-flex-box': FtFlexBox,
+    'ft-loader': FtLoader
+  },
+  data: function() {
+    return {
+      hashtag: '',
+      hashtagContinuationData: null,
+      videos: [],
+      apiUsed: 'local',
+      isLoading: true
+    }
+  },
+  computed: {
+    backendPreference: function () {
+      return this.$store.getters.getBackendPreference
+    },
+
+    backendFallback: function () {
+      return this.$store.getters.getBackendFallback
+    },
+
+    showFetchMoreButton() {
+      return !isNullOrEmpty(this.hashtagContinuationData)
+    }
+  },
+  watch: {
+    $route() {
+      this.isLoading = true
+      this.hashtag = ''
+      this.hashtagContinuationData = null
+      this.videos = []
+      this.apiUsed = 'local'
+      this.getHashtag()
+    }
+  },
+  mounted: async function() {
+    this.isLoading = true
+    await this.getHashtag()
+  },
```

Comment:
I don't think Vue 2 supports async lifecycle hooks, so it won't `await` this hook, so it doesn't need to be async with await
```suggestion
  mounted: function() {
    this.isLoading = true
    this.getHashtag()
  },
```